project/BinaryIncompatibilities.scala

```diff
@@ -6,6 +6,40 @@ object BinaryIncompatibilities {
   )
 
   val Tools = Seq(
+      ProblemFilters.exclude[MissingMethodProblem](
+        "org.scalajs.core.tools.linker.backend.emitter.ScalaJSClassEmitter.this"),
```

Comment:
`ScalaJSClassEmitter` *itself* is `private[scalajs]`, so all its methods are "implicitly" `private[scalajs]` too.